sdk/core/core-amqp/src/util/hmacSha256.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
could we reuse functions from core-util

https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/core/core-util/src/sha256.ts

Reply:
They are a bit different in subtle ways.  For now, we can leave these as is in core-amqp

Reply:
maybe open a tracking issue to look at unification?